bin/polyserve

```diff
@@ -24,6 +24,7 @@ Options:
   -b <browsername> use this browser instead of default
                    (ex: 'Google Chrome Canary')
   -H <hostname>    use this hostname instead of localhost
+  --cors           enable permissive cors headers in server responses
```

Comment:
I can change it. The utility seems limited to me, though, since the feature is opt-in to begin with.

If we change it, perhaps the option should be more explicitly named (e.g., `--allow-origin`). WDYT?